packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -170,6 +177,8 @@ function AuthOptions({
   );
   const { refetchBoot, user } = useAuthContext();
   const router = useRouter();
+  const isOnboardingPage = !!router?.pathname?.startsWith('/onboarding');
```

Comment:
Where do we still show it that's not onboarding?
Didn't we move everything to uniform flow?

Reply:
Currently we still have an auth banner shown when guest user view a post page